contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -55,22 +60,22 @@ def span_data_to_envelope(self, sd):
             tags=dict(utils.azure_monitor_context),
             time=sd.start_time,
         )
+
         envelope.tags['ai.operation.id'] = sd.context.trace_id
         if sd.parent_span_id:
-            envelope.tags['ai.operation.parentId'] = '|{}.{}.'.format(
-                sd.context.trace_id,
+            envelope.tags['ai.operation.parentId'] = '{}'.format(
                 sd.parent_span_id,
             )
         if sd.span_kind == SpanKind.SERVER:
             envelope.name = 'Microsoft.ApplicationInsights.Request'
             data = Request(
-                id='|{}.{}.'.format(sd.context.trace_id, sd.span_id),
+                id='{}'.format(sd.span_id),
                 duration=utils.timestamp_to_duration(
                     sd.start_time,
                     sd.end_time,
                 ),
-                responseCode='0',
-                success=False,
+                responseCode=str(sd.status.code),
+                success=False,  # Modify based off attributes or status
                 properties={},
```

Comment:
this was not listed as a bug, but I noticed we don't populate properties for HTTP requests. Can we do it?

request.name = "METHOD /route" (if route is not populated, then path)
request.url = url